### ISSUE TITLE
Month/Year selects won't close

### ISSUE THREAD

**Issue by shoxty** [voidfire/calendar#5]
I am having an issue where the month and year selects won't close after they have been opened. When you try and close them they both throw the same error "Error: A DOM element reference is required". Do you have any idea what this could be or how I might be able to debug it?

I found this post of someone having the same issue:

[https://github.com/laravel/nova-issues/issues/1647](https://github.com/laravel/nova-issues/issues/1647)

**Comment by voidfire** [comment]
I believe it is related to https://github.com/ant-design/ant-design/issues/10269
I can replicate in my nova/calendar setup but everything seem to work despite the error..I'll dig a bit more

**Comment by shoxty** [comment]
So you can get the error to display but your month/year selects close?

**Comment by voidfire** [comment]
Yes, in two projects I have calendar showing the errors about DOM reference required when I click the selects. but they are functional
In these projects the difference is that the package is not loaded through / packagist
But it is included in nova-components and in composer json loaded in the same way as nova with path repository like this
![image](https://user-images.githubusercontent.com/4143062/82139897-37cb1980-9834-11ea-91fb-dcd4a2c695b4.png)

You could try that as a workaround until we dig deeper into the vuejs reference webpack issues that Antd has
